apis/sgv2-quarkus-common/src/main/resources/application.yaml

```diff
@@ -152,3 +152,15 @@ quarkus:
               datastore:
                 configuration:
                   DataStorePropertiesConfiguration/fetchSupportedFeatures/Retry/maxRetries: 0
+
+# shared production properties
+"%prod":
+
+  quarkus:
+
+    # OpenTelemetry configuration
+    opentelemetry:
```

Comment:
agreed to disable open telemetry by default in production @kathirsvn

Reply:
How is it disabled here? Isn't this just setting sampler, but leaving it enabled (same as before change)?

Reply:
It's not, we want to do it by default.. For now in prod it's disabled..

Reply:
Actually, I can configure this without a `prod` profile at all..

Reply:
Ah. I like this, cool.